scripts/launch-dev-node-with-postgres

```diff
@@ -11,8 +11,22 @@ export POSTGRES_PASSWORD=$ESPRESSO_SEQUENCER_POSTGRES_PASSWORD
 
 export RUST_LOG=${RUST_LOG:-info}
 
+# Function to handle SIGTERM and SIGINT
+cleanup() {
+    echo "Received SIGTERM or SIGINT. Exiting..."
+    kill -TERM "$postgres_pid" 2>/dev/null
+    kill -TERM "$dev_node_pid" 2>/dev/null
+    wait "$postgres_pid"
```

Comment:
We can use `jobs -p` to get a list of all background processes, then we don't need to track PIDs explicitly.

There are also some oneliners in this SO answer: https://stackoverflow.com/a/2173421